sdk/ai/azure-ai-inference/tests/test_prompts.py

```diff
@@ -0,0 +1,92 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import os
+from azure.ai.inference.prompts import PromptTemplate
+from devtools_testutils import AzureRecordedTestCase
+
+
+class TestPrompts(AzureRecordedTestCase):
```

Comment:
Do you need the base class here? Try to remove it, since you are not doing any network traffic recordings

Reply:
Thanks, let me remove it